src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160425.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="hussein" id="temp_table_for_expr_evaluation">
+            <sql stripComments="true">
+                create temporary table EXPREVAL_TEMP ( name VARCHAR(254) NOT NULL );
```

Comment:
So yeah. I had to add a new table. `EXPREVAL_TEMP` stores the names of the batch of entities we're evaluating expressions against. This allows us to keep track of the "root" entity for each attribute returned by the db query, since that information is no longer implied by the callstack.